routes/candidates_routes.js

```diff
@@ -1,15 +1,27 @@
 const express = require('express')
+const bodyparser = require('body-parser')
+const mongodb = require('mongodb')
 
 const { database } = require('../mongo')
 const c_router = express.Router()
 
 c_router
-    .get('/:candidateId',function(req, res, next) {
-        let a1 =req.method
-        let a2 = req.params
-        res.send(`uri is : ${a2} \n method is : ${a1} `)
-        console.log(database.db)
-     }) 
+    .get('/:candidateId', async function(req, res, next) {
+    // let a1 =req.method
+    // let a2 = req.params
+    //     res.send(`uri is : ${a2} \n method is : ${a1} `)
+    //    // console.log(database.db)
+        const collection = database.db.collection("candidate")
+        const cur = collection.find({})
+        
+        let item = []
+        await cur.forEach(function (doc) {
+            item.push(doc)
+        })
```

Comment:
Here you are:
1) Creating an empty array
2) Iterating over a cursor from the database
3) Pushing these items to the array
---
But you could just simple await on find() and get all items at once with
```
const items = await collection.find({})
```